static/app/views/alerts/rules/metric/details/metricHistory.tsx

```diff
@@ -1,93 +1,168 @@
-import {useMemo} from 'react';
+import {Fragment} from 'react';
 import {css} from '@emotion/react';
 import styled from '@emotion/styled';
+import moment from 'moment-timezone';
 
 import CollapsePanel from 'sentry/components/collapsePanel';
+import {DateTime} from 'sentry/components/dateTime';
+import Duration from 'sentry/components/duration';
+import Link from 'sentry/components/links/link';
 import {PanelTable} from 'sentry/components/panels/panelTable';
+import {StatusIndicator} from 'sentry/components/statusIndicator';
 import {t, tn} from 'sentry/locale';
 import {space} from 'sentry/styles/space';
-import type {ActivationTriggerActivity, AlertRuleActivation} from 'sentry/types/alerts';
-// import {ActivationTrigger} from 'sentry/types/alerts';
+import type {Organization} from 'sentry/types/organization';
+import getDuration from 'sentry/utils/duration/getDuration';
+import getDynamicText from 'sentry/utils/getDynamicText';
+import {capitalize} from 'sentry/utils/string/capitalize';
 import useOrganization from 'sentry/utils/useOrganization';
-import MetricAlertActivity from 'sentry/views/alerts/rules/metric/details/metricActivity';
-import MetricHistoryActivation from 'sentry/views/alerts/rules/metric/details/metricHistoryActivation';
-import type {Incident} from 'sentry/views/alerts/types';
+import {COMPARISON_DELTA_OPTIONS} from 'sentry/views/alerts/rules/metric/constants';
+import {AlertRuleThresholdType} from 'sentry/views/alerts/rules/metric/types';
+import type {ActivityType, Incident} from 'sentry/views/alerts/types';
+import {IncidentActivityType, IncidentStatus} from 'sentry/views/alerts/types';
+import {alertDetailsLink} from 'sentry/views/alerts/utils';
+import {AlertWizardAlertNames} from 'sentry/views/alerts/wizard/options';
+import {getAlertTypeFromAggregateDataset} from 'sentry/views/alerts/wizard/utils';
 
 const COLLAPSE_COUNT = 3;
 
+type MetricAlertActivityProps = {
```

Comment:
There were a lot of changes to this file for activated alerts so I went back to how the file was before the 2 commits for activated alerts and copied that all in - https://github.com/getsentry/sentry/blame/0c692d54aaa5aecb5e02d346657d2d1f11ffcf2f/static/app/views/alerts/rules/metric/details/metricHistory.tsx

Reply:
👍 i like that approach, probably a lot safer than us trying to carve it out.